### PR TITLE
doc/storage: move content from FAQ to storage section

### PR DESCRIPTION
**Pull request by ru-fu**
Move some details from the FAQ about creating storage to the
storage section and remove the question from the FAQ.

Signed-off-by: Ruth Fuchss <ruth.fuchss@canonical.com>

### CODE REVIEW

File: doc/howto/storage_create_pool.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ To create a storage pool, use the following command:
 
     lxc storage create <pool_name> <driver> [configuration_options...]
 
+Unless specified otherwise, LXD sets up loop-based storage with a sane default size (20% of the free disk space, but at least 5 GiB and at most 30 GiB).
```
Comment (tomponline) [inline doc/howto/storage_create_pool.md:11]: I believe "sane" is frowned upon for inclusive language, as to relates to mental health (the counter being insane).

File: doc/explanation/storage.md [hunk 2]
```diff
@@ -18,6 +18,9 @@ The following storage drivers are supported:
 - {ref}`storage-ceph`
 - {ref}`storage-cephfs`
 
+During initialization, you can create either a directory-based storage pool or a ZFS storage pool.
```
Comment (tomponline) [inline doc/explanation/storage.md:21]: I'm not sure thats true, will need to check...

File: doc/howto/storage_create_pool.md (hunk 1 above)
Comment (ru-fu) [reply doc/howto/storage_create_pool.md:11]: Oh right - I wonder why the inclusive language check doesn't catch it. Probably that one isn't in the official word list yet ...

File: doc/explanation/storage.md (hunk 2 above)
Comment (ru-fu) [reply doc/explanation/storage.md:21]: Darn, I checked whether it was "can" or "must", but stopped right after that (and released the VM).

Comment (tomponline) [reply doc/explanation/storage.md:21]: Confirmed:

```
lxd init
Would you like to use LXD clustering? (yes/no) [default=no]: 
Do you want to configure a new storage pool? (yes/no) [default=yes]: 
Name of the new storage pool [default=default]: 
Name of the storage backend to use (ceph, btrfs, dir, lvm, zfs) [default=zfs]: 
```

Comment (stgraber) [reply doc/explanation/storage.md:21]: `lxd init` lets you use all the different types except for cephfs.

```
stgraber@dakara:~/data/code/lxc/linuxcontainers.org (lxc/master)$ lxd init
Would you like to use LXD clustering? (yes/no) [default=no]: 
Do you want to configure a new storage pool? (yes/no) [default=yes]: 
Name of the new storage pool [default=default]: blah
Name of the storage backend to use (zfs, ceph, btrfs, dir, lvm) [default=zfs]: 
```

The logic for the default is somewhat complex but roughly it is:
 - Use zfs if the system is using zfs for its root
 - Use btrfs if the system is using btrfs for its root
 - Use zfs loop if zfs kernel support is detected
 - Use the dir backend

Comment (ru-fu) [reply doc/explanation/storage.md]: Thanks for checking! Updated.

**Review by tomponline — APPROVED** [review]

**Pull request merged**